config/config.go

```diff
@@ -470,6 +475,8 @@ func loadConfig(logger *logrus.Entry, path string, mandatory bool) (rCfg *Config
 		}
 	}
 
+	cfg.CustomDNS.Zone.configPath = prettyPath
```

Comment:
Could you move the two non-absolute `$INCLUDE` tests to `config_test.go` and change them to parse the config from YAML instead of creating a `ZoneFileDNS` manually in Go?  
That way we're testing the whole flow of the feature.

There's already a bunch of tests in there that create files we run through the whole config parsing pipeline.  
One example:
```go
It("should ignore non YAML files", func() {
	tmpDir.CreateStringFile("ignore-me.txt", "THIS SHOULD BE IGNORED!")

	_, err := LoadConfig(tmpDir.Path, true)
	Expect(err).Should(Succeed())
})
```

Reply:
This was a good move. It showed that we hadn't accounted for cases where blocky is launched with a directory as the config parameter. `blocky -c ./config/` would have previously failed :sweat_smile:

Reply:
Nice, quick payoff!